gke/k8sresources.tf

```diff
@@ -1,39 +1,12 @@
 provider "kubernetes" {
+  version = "~> 1.10.0"
   load_config_file = false
   host = "https://${google_container_cluster.gke-cluster.endpoint}"
   version                = "1.10.0"
```

Comment:
```suggestion
```

Reply:
Terraform fails because version is defined twice

Reply:
Yes, sorry - I think I didn't resolve the merge conflict correctly. Anyway, have pushed the fixes.